apps/backend/src/resources/spec/GLRecord.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { GLRecord } from '../';
+import { SalesDTO } from '../../sales/dto/sales.dto';
+
+const salesRecord = require('./fixtures/sale.json');
+describe('GLRecord', () => {
+  it('transform from sales event ', () => {
+    const glRecord = GLRecord.transformSalesEvent([salesRecord as SalesDTO]);
+    console.log(glRecord.batchHeader.convertFromJson().toString());
```

Comment:
Is this log intended for development only? Should we use the custom logger here instead of console?